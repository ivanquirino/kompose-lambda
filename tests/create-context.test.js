const createContext = require('../lib/create-context')

test('create context', () => {
  const ctx = createContext()

  expect(ctx).toMatchSnapshot()
})
