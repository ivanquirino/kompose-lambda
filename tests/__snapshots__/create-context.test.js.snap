// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create context 1`] = `
Object {
  "callback": [Function],
  "context": Object {},
  "customContext": Object {},
  "event": Object {},
  "result": Object {
    "body": "",
    "headers": Object {},
    "isBase64Encoded": false,
    "statusCode": null,
  },
}
`;
